src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -36,8 +36,18 @@ export const applyForAccess = async (selected, history) => {
     const draftResponse = await DAR.postDarDraft({ datasetId: selected });
     if (draftResponse.referenceId) {
       history.push(`/dar_application/${draftResponse.referenceId}`);
-    } else if (draftResponse.message) {
-      Notifications.showError({ text: draftResponse.message + ' Please contact customer support for help.' });
+    } else if (draftResponse.code && draftResponse.code === 422) {
+      Notifications.showError(
+          {
+            text: "Please register in DUOS with your institution's email and obtain approval from your Signing Official in order to submit a data access request.",
+            severity: 'error',
+            timeout: 6000,
+            layout: {
+              vertical: 'bottom',
+              horizontal: 'right'
+            }
```

Comment:
I made some recent changes so that most of the properties (severity, timeout, layout) are now optional: https://github.com/DataBiosphere/duos-ui/blob/657c44527131a6408e23f699c9936d8afbfac893/src/libs/ToastNotifications.tsx#L16